backend/WebApi/Controllers/ModificationEntryController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using ModelLayer.DataTransferObjects;
+using ModelLayer.Models;
+using NSwag.Annotations;
+using ServiceLayer;
+
+namespace WebApi.Controllers
+{
+    [Route("api/home")]
+    [ApiController]
+    public class ModificationEntryController : ControllerBase
+    {
+        private readonly IMapper _mapper;
+        private readonly IModificationEntryService modificationEntryService;
+
+        public ModificationEntryController(IMapper mapper, IModificationEntryService modificationEntryService)
+        {
+            this._mapper = mapper;
```

Comment:
this operator ist hier überflüssig
-> entweder mit "this" operator oder mit "_" anführend vor private readonly Objekten

-> ich bevorzuge "_" anführend zu machen

Reply:
ok